inspector/src/components/actionTabs/actionTabsComponent.tsx

```diff
@@ -24,9 +24,20 @@ interface IActionTabsComponentProps {
     popupMode?: boolean,
     onPopup?: () => void,
     onClose?: () => void,
-    globalState?: GlobalState
+    globalState?: GlobalState,
+    initialTab?: string

```

Comment:
Why not relying on an enum ?

Reply:
i found that /src/tsconfig.json restricted 'rootDir' to './', implied that there's no way to 
> export an `enum` from /inspector/src/..

and then 

> import it to /src/Debug/debugLayer.ts 

so i chose to use `string`